### PR TITLE
Add container apps hosting option

### LINKED ISSUE #1952: Add docs for Azure Container Apps
Azure Container Apps now has first-class support for Dapr: https://docs.microsoft.com/en-us/azure/container-apps/microservices-dapr?tabs=bash

We should add a hosting option listing Container Apps. Additionally any other offerings that support Dapr should also be added in the future.

### PR DESCRIPTION
**Pull request by AaronCrawfis**

### CODE REVIEW

File: daprdocs/content/en/operations/hosting/azure-container-apps/index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+type: docs
+title: "Run Dapr in Azure Container Apps"
+linkTitle: "Azure Container Apps"
+weight: 3000
+description: "Learn how to run your Dapr apps on the Azure Container Apps serverless runtime"
+---
+
+[Azure Container Apps](https://docs.microsoft.com/en-us/azure/container-apps/overview) is a serverless application centric hosting service where users do not see or manage any underlying VMs, orchestrators, or other cloud infrastructure. Azure Container Apps enables executing application code packaged in any container and is unopinionated about runtime or programming model.
```
Comment (willtsai) [inline daprdocs/content/en/operations/hosting/azure-container-apps/index.md]:
```suggestion
[Azure Container Apps](https://docs.microsoft.com/en-us/azure/container-apps/overview) is a serverless application centric hosting service where users do not see nor manage any underlying VMs, orchestrators, or other cloud infrastructure. Azure Container Apps enables executing application code packaged in any container and is unopinionated about runtime or programming model.

```

File: daprdocs/content/en/operations/hosting/azure-container-apps/index.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+[Azure Container Apps](https://docs.microsoft.com/en-us/azure/container-apps/overview) is a serverless application centric hosting service where users do not see or manage any underlying VMs, orchestrators, or other cloud infrastructure. Azure Container Apps enables executing application code packaged in any container and is unopinionated about runtime or programming model.
+
+Dapr is built-in as a first-class experience in Container Apps, letting you use Dapr building blocks without any manual deployment or management of Kubernetes or the Dapr runtime. Simply deploy your services and Dapr components and let the platform take care of the rest.
+
+{{< button text="Learn more" link="https://docs.microsoft.com/en-us/azure/container-apps/overview" newtab="true" >}}
+
+## Tutorial
+
+Visit the [Azure docs](https://docs.microsoft.com/en-us/azure/container-apps/microservices-dapr) to try out a microservices tutorial, where you'll model two services and add Dapr:
```
Comment (willtsai) [inline daprdocs/content/en/operations/hosting/azure-container-apps/index.md]:
```suggestion
Visit the [Azure docs](https://docs.microsoft.com/en-us/azure/container-apps/microservices-dapr) to try out a microservices tutorial, where you'll deploy two Dapr-enabled services in Azure Container Apps:

```

**Review by willtsai — COMMENTED** [review]
some minor wordsmithing, feel free to incorporate as you see fit

File: daprdocs/content/en/operations/hosting/azure-container-apps/index.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+title: "Run Dapr in Azure Container Apps"
+linkTitle: "Azure Container Apps"
+weight: 3000
+description: "Learn how to run your Dapr apps on the Azure Container Apps serverless runtime"
+---
+
+[Azure Container Apps](https://docs.microsoft.com/en-us/azure/container-apps/overview) is a serverless application centric hosting service where users do not see nor manage any underlying VMs, orchestrators, or other cloud infrastructure. Azure Container Apps enables executing application code packaged in any container and is unopinionated about runtime or programming model.
+
+
+Dapr is built-in as a first-class experience in Container Apps, letting you use Dapr building blocks without any manual deployment or management of Kubernetes or the Dapr runtime. Simply deploy your services and Dapr components and let the platform take care of the rest.
```
Comment (msfussell) [inline daprdocs/content/en/operations/hosting/azure-container-apps/index.md]:
```suggestion
Dapr is built-in to Container Apps, enabling you to use the Dapr API building blocks without any manual deployment of the Dapr runtime. You simply deploy your services with their Dapr components.

```

File: daprdocs/content/en/operations/hosting/azure-container-apps/index.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
+type: docs
+title: "Run Dapr in Azure Container Apps"
+linkTitle: "Azure Container Apps"
+weight: 3000
+description: "Learn how to run your Dapr apps on the Azure Container Apps serverless runtime"
+---
+
+[Azure Container Apps](https://docs.microsoft.com/en-us/azure/container-apps/overview) is a serverless application centric hosting service where users do not see nor manage any underlying VMs, orchestrators, or other cloud infrastructure. Azure Container Apps enables executing application code packaged in any container and is unopinionated about runtime or programming model.
```
Comment (msfussell) [inline daprdocs/content/en/operations/hosting/azure-container-apps/index.md]:
```suggestion
[Azure Container Apps](https://docs.microsoft.com/en-us/azure/container-apps/overview) is a serverless application hosting service where users do not see nor manage any underlying VMs, orchestrators, or other cloud infrastructure. Azure Container Apps enables you to run your application code packaged in multiple containers and is unopinionated about runtimes or programming models that are used.
```

File: daprdocs/content/en/operations/hosting/azure-container-apps/index.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+---
+type: docs
+title: "Run Dapr in Azure Container Apps"
+linkTitle: "Azure Container Apps"
+weight: 3000
+description: "Learn how to run your Dapr apps on the Azure Container Apps serverless runtime"
```
Comment (msfussell) [inline daprdocs/content/en/operations/hosting/azure-container-apps/index.md]:
```suggestion
description: "Learn how to run your Dapr applications on the Azure Container Apps serverless platform "

```

File: daprdocs/content/en/operations/hosting/azure-container-apps/index.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+[Azure Container Apps](https://docs.microsoft.com/en-us/azure/container-apps/overview) is a serverless application centric hosting service where users do not see nor manage any underlying VMs, orchestrators, or other cloud infrastructure. Azure Container Apps enables executing application code packaged in any container and is unopinionated about runtime or programming model.
+
+
+Dapr is built-in as a first-class experience in Container Apps, letting you use Dapr building blocks without any manual deployment or management of Kubernetes or the Dapr runtime. Simply deploy your services and Dapr components and let the platform take care of the rest.
+
+{{< button text="Learn more" link="https://docs.microsoft.com/en-us/azure/container-apps/overview" newtab="true" >}}
+
+## Tutorial
+
+Visit the [Azure docs](https://docs.microsoft.com/en-us/azure/container-apps/microservices-dapr) to try out a microservices tutorial, where you'll deploy two Dapr-enabled services in Azure Container Apps:
```
Comment (msfussell) [inline daprdocs/content/en/operations/hosting/azure-container-apps/index.md]:
```suggestion
Visit the [Azure docs](https://docs.microsoft.com/en-us/azure/container-apps/microservices-dapr) to try out a microservices tutorial, where you'll deploy two Dapr-enabled applications to Azure Container Apps.

```

**Review by greenie-msft — APPROVED** [review]

**Pull request merged**